assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -469,6 +470,30 @@ export const selectors = {
 		return queryArg ? queryArg : false;
 	},
 
+	/**
+	 * Gets external help links which includes the user's locale.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state        Data store's state.
+	 * @param {Object} [args]       Optional arguments for the resulting URL.
+	 * @param {string} [args.path]  Base URL to build complete URL.
+	 * @param {Object} [args.query] Object to append query to the URL.
+	 * @param {string} [args.hash]  Optional hash.
+	 * @return {(string|null)} The URL containing the user's locale or `null` if path is not set.
+	 */
+	getGoogleSupportURL: ( state, args ) => {
+		const { path, query, hash } = args || {};
+
+		if ( ! path ) {
+			return null;
+		}
+
+		const url = new URL( addQueryArgs( path, { ...query, hl: getLocale() } ) );
```

Comment:
I didn't see the exact IB for this one before (which this matches), but I was thinking the `path` should be an actual path, not a full URL. The `getGoogleSupportURL` selector should always return URLs starting in `https://support.google.com`, and only the other parts (`path`, `query`, and `hash`) would be passed to the selector.

It's confusing to provide a full URL as an argument called `path` IMO.

Reply:
@felixarntz Code updated. Can you check?